app.js

```diff
@@ -44,12 +45,20 @@ const port = settings && settings.port ? settings.port : 3005
 
 if (!module.parent) {
   // application entry point, called directly
-  app.listen(port, host, function (error) {
-    if (error != null) {
-      throw error
-    }
-    return logger.info(`spelling starting up, listening on ${host}:${port}`)
-  })
+  mongodb
+    .waitForDb()
+    .then(() => {
+      app.listen(port, host, function (error) {
+        if (error != null) {
+          throw error
+        }
+        return logger.info(`spelling starting up, listening on ${host}:${port}`)
```

Comment:
Unnecessary return?

Reply:
Yes. There are quite a lot of others too. I just changed the indent only here -- or the collection method in other places. The spelling service is due for a general decaff-cleanup -- or removal after all, hint front end spell-check hint, hint :)